gimvi_tutorial.ipynb

```diff
@@ -9,7 +9,7 @@
    },
```

Comment:
<p>should the increment batch_idx be happening inside gimvi?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
hidden